demo-circuit/src/constants/mod.rs

```diff
@@ -266,4 +287,42 @@ mod test {
         println!("{:#?}", htc_g2_out);
         assert_eq!(gh_generators, VRFParams::new().group_hash_generators);
     }
+
+    #[serial]
+    #[test]
+    fn test_csw_phantom_public_key() {
+        let x = SimulatedFieldElement::from(BigInteger256([
+            15877199453377760308,
+            458271239891050623,
+            5539075294202620951,
+            5404726604943382876,
+        ]));
+        let y = SimulatedFieldElement::from(BigInteger256([
+            3725370832501899225,
+            12695286482625208691,
+            7386704395789842856,
+            3479569230309726808,
+        ]));
+        let simulated_te_point = SimulatedTEGroup::new(x, y);
+        assert_eq!(simulated_te_point.group_membership_test(), false);
```

Comment:
Actually, it is not obvious that our `hash_to_curve` outputs a point outside the prime order subgroup, given the specific `Strontium` string. I would remove this assertion, as it does not play any role for security.